packages/main/src/MultiInput.ts

```diff
@@ -213,15 +198,13 @@ class MultiInput extends Input {
 
 		const target = e.target as HTMLInputElement;
 		const isHomeInBeginning = isHome(e) && target.selectionStart === 0;
-		const isCtrl: boolean = e.metaKey || e.ctrlKey;
-		const tokens = this.tokens;
 
 		if (isHomeInBeginning) {
 			this._skipOpenSuggestions = true; // Prevent input focus when navigating through the tokens
 			return this._focusFirstToken(e);
 		}
 
-		if (isLeft(e) || isBackSpace(e)) {
+		if (isLeft(e) || isUp(e) || isBackSpace(e)) {
```

Comment:
Is it stated somewhere that when the focus is in the input field and up is pressed the focus should go to the tokenizer?
I checked:
- the tokenizer spec
- multi input spec
- keyboard handling specification 
I do not see such interaction described, what is more for the MultiInput in the KBH spec it is stated that if the focus is the input field on arrow up nothing should happen.
What is more, this is working only on a closed suggestions list (since there we have a flat navigation through the list of suggestions/VSH and input field).

Reply:
another point regarding this is that this will differentiate  the interaction in the MCB with the one in the multi input, which in my opinion would bring up confusion to the end user, rather than making the control more usable